admin/formatter/class-post-metabox-formatter.php

```diff
@@ -105,16 +105,54 @@ private function base_url_for_js() {
 	}
 
 	/**
-	 * Counting the number of given keyword used for other posts than given post_id
+	 * Counts the number of given keywords used for other posts other than the given post_id.
 	 *
-	 * @return array
+	 * @return array The keyword and the associated posts that use it.
 	 */
 	private function get_focus_keyword_usage() {
 		$keyword = WPSEO_Meta::get_value( 'focuskw', $this->post->ID );
+		$usage   = array( $keyword => $this->get_keyword_usage_for_current_post( $keyword ) );
 
-		return array(
-			$keyword => WPSEO_Meta::keyword_usage( $keyword, $this->post->ID ),
-		);
+		if ( WPSEO_Utils::is_yoast_seo_premium() ) {
```

Comment:
I think it is nicer to have a filter for this? So we can hook into it from Premium.

Reply:
This is for internal use only. Using a filter would mean people could hook into it and break the query altogether.